docs/actions.rst

```diff
@@ -51,6 +44,9 @@ All actions in Crud, even your own, should inherit from the
 This class is ``abstract`` and provides numerous auxiliary methods which can be
 useful for you both as a developer as an action creator.
 
+Where you choose to put your custom Crud actions is up to you, but using ``src/Crud`` is a good place. Remember to
```

Comment:
I don't know either but I do know I don't like this escaping shizzle, it works fine when not inside the Crud namespace. Any core-dev available to shed some light on this?